### PR TITLE
A couple of fixes.

### PR DESCRIPTION
**Pull request by mzhurovich**
Result of global `make test` on my Mac:
```
[----------] Global test environment tear-down
[==========] 267 tests from 46 test cases ran. (8763 ms total)
[  PASSED  ] 267 tests.

ALL TESTS PASS.
```

### CODE REVIEW

File: Storage/storage.h [hunk 1]
```diff
@@ -75,26 +75,6 @@ namespace storage {
 // 2) Splits the type into `T_ADDER` and `T_DELETER`, to support seamless persistence of deletions.
 // clang-format off
 
-#ifndef _MSC_VER
```
Comment (dkorolev) [inline Storage/storage.h]: Does this work in Visual Studio?

**Comment by dkorolev** [comment]
@mzhurovich Yay!

LGTM++ and good to merge if it doesn't break Windows.

Thanks!

**Comment by mzhurovich** [comment]
@dkorolev Sorry, didn't see your pull-request. Fixing the mess now...

File: Storage/storage.h (hunk 1 above)
Comment (mzhurovich) [inline Storage/storage.h]: My last change to `struct.h` unified the base-class constructor call under `: SUPER(...)`. Actually, you first made this change for MSVC, so it must work there :)

Comment (dkorolev) [inline Storage/storage.h]: Yes. I'm just reluctant to merge multiple changes in knowing there will be
more changes, and I won't see Windows for the next two days. Fixing it back
later, even if it's a one line change, would be a pain.

Comment (mzhurovich) [inline Storage/storage.h]: BTW you made exactly the same change here:
https://github.com/dkorolev/Current/commit/f3756135e5539592556ffaa53477e74e9dcf6038
)))

Comment (dkorolev) [inline Storage/storage.h]: Ah, then it's probably fine )))

**Comment by mzhurovich** [comment]
@dkorolev PTAL once more. Works fine on my Mac.

**Comment by dkorolev** [comment]
Compiling on Ubuntu ...

**Comment by dkorolev** [comment]
Green!

**Pull request merged**